Doc/user_manual/six.tex

```diff
@@ -1519,9 +1519,9 @@ \subsection{Electron lens} \label{elen}
 \paragraph{Keyword} ELEN
 \paragraph{Number of data lines} variable
 \paragraph{Format} \emph{name type thetamax r2 r2ovr1 offset\_x offset\_y flag\_entrance flag\_exit}
-A description of the input parameters for the different e-lens types is given in Table~\ref{ELEN}. Currently only the ideal hollow electron lens is implemented in SixTrack (type ANNULAR). 
+A description of the input parameters for the different e-lens types is given in Table~\ref{elen}. Currently only the ideal hollow electron lens is implemented in SixTrack (type ANNULAR). 
```

Comment:
The table is named `tab:ELEN`, the paragraph  is named `elen`.

Suggestion:
Rename the section from `elen` -> `sec:elen`, the table `tab:ELEN`->`tab:elen`, and update you references.